nomic/atlas.py

```diff
@@ -77,13 +77,19 @@ def map_embeddings(
     if description:
         description = description
 
+    added_id_field = False
     if data is None:
         data = [{ATLAS_DEFAULT_ID_FIELD: b64int(i)} for i in range(len(embeddings))]
+        added_id_field = True
 
     if id_field == ATLAS_DEFAULT_ID_FIELD and id_field not in data[0]:
+        added_id_field = True
         for i in range(len(data)):
             data[i][id_field] = b64int(i)
 
+    if added_id_field:
+        logger.warning("An ID field was not specified in your data so one was generated for you..")
```

Comment:
Should we mention the ID is in order from `range(len(data))` in order that the data was added?

Reply:
updated